mergify_engine/actions/merge_base.py

```diff
@@ -54,13 +55,37 @@ class PriorityAliases(enum.Enum):
     high = 3000
 
 
+class StrictMergeParameter(enum.Enum):
+    true = True
+    false = False
+    fasttrack = "smart+fasttrack"
+    ordered = "smart+ordered"
+
+
+if StrictMergeParameter.__name__ not in mergify_json.JSON_TYPES:
```

Comment:
That check is more likely to be done by `register_type`. It'd be stupid to have each caller of `register_type` doing this check!